### ISSUE TITLE
how to deal with manual encryption and then adding non autocrypt recipient

### ISSUE THREAD

**Issue by azul** [autocrypt/autocrypt#104]
(added from a todo in 'Provide a recommendation for message encryption')

```
564 .. todo::
565 
566    Discuss how to deal with the case where the user manually selects
567    encryption and subsequently adds a recipient whom the MUA has no
568    key.
```

**Comment by r10s** [comment]
In general, I would leave the encryption Icon enabled; maybe add a "little red x" (hello @Valodim ) to the icon showing that something is not 100% perfect.

Maybe the user has added a wrong address (private, work, ..) and changed it to another one that can be used for encryption. Or he's just added someone on error. 

When the user finally has made its settings and then clicks onto "Send" and there are keys missing for some recipient, I would show an error dialog that the message cannot be send with these settings. I would _not_ ask in this dialog something like "cannot encrypt - send anyway - yes - no" - chances are great this is clicked away before read.

_Shorter:_ If the user has manually enabled encryption, he should also manually disable it again.

**Comment by dkg** [comment]
@Valodim resolved this with 6f3a6750bda6b800d681c171c5c31bb00cdbb38f
